Opensourcecommunity/Getstarted.md

```diff
@@ -0,0 +1,7 @@
+This is guidance to start contributing to the open smart grid platform project:

```

Comment:
Maybe change 'This is guidance' into 'This is a guide to start contributing ...' 

or 'Please follow the steps below to start contributing..'